src/mocks/data/draft_reports_object.json

```diff
@@ -0,0 +1,50 @@
+{
+  "status": "processing", // enum: one of ["processing", "failed", "delivered"]
+  "current_stage": "dex-upload",
+  "current_action": "metadata-validation",
+  "file_name": "test.txt",
+  "transport_id": "unique_guid", //represents the id for the entire transaction i.e. upload_id
+  "timestamp": "12-12-2023", -- first initial timestamp
+  "data_stream_id": "aims-celr",
+  "data_stream_route": "csv",
+  "reports": [
+    {
+    "stage": "DEX HL7 v2", //describes a chunk of the pipeline, usually associated with a specific product; string could be human readable, or transformed from id to human readable
+    "action": "debatch", //describes specific thing that happened as part of the stage
+    "schema_name": "dex-hl7",
+    "version": "1.0",
+    "data_stream_id": "aims-celr",
+    "data_stream_route": "csv",
+    "jurisdiction": "MD",
+    "transport_id": "unique_guid", //represents the id for the entire transaction i.e. upload_id
+    "parent_id": "unique_guid", (would be null if it is the parent?) //id for the thing that resulted in this report?
+    "message_id": "unique_guid",//id for the thing this report is about
```

Comment:
message_id is a start - HL7 collects these metadata fields in regards to message:

"message_metadata":{
    "message_uuid":"08b1184b-cad8-4f4a-9f32-09e64b854713",
    "single_or_batch":"SINGLE",
    "message_index":1,
    "message_hash":"1bd53942a07b2e4712a02f4e79e3a9cd"}

Reply:
they definitely apply to CSV. they can potentially apply to FHIR, but will be better to get a FHIR SMEs. Will they be sending batches of FHIR messages in a single file?